src/parser/priest/discipline/modules/shadowlands/conduits/ShiningRadiance.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import calculateEffectiveHealing from 'parser/core/calculateEffectiveHealing';
+import Events, { HealEvent } from 'parser/core/Events';
+import { Options } from 'parser/core/Module';
+import { SHINING_RADIANCE_INCREASE } from 'parser/priest/discipline/constants';
+
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import ConduitSpellText from 'interface/statistics/components/ConduitSpellText';
+import ItemHealingDone from 'interface/ItemHealingDone';
+
+class ShiningRadiance extends Analyzer {
+
+  conduitRank: number = 0;
+  bonusRadianceHealing: number = 0;
+
+  constructor(options: Options) {
+    super(options);
+
+    this.conduitRank = this.selectedCombatant.conduitRankBySpellID(SPELLS.SHINING_RADIANCE.id);
+    if (!this.conduitRank) {
+      this.active = false;
+      return;
+    }
+
+    this.addEventListener(Events.heal.by(SELECTED_PLAYER).spell(SPELLS.POWER_WORD_RADIANCE), this.onRadianceHeal);
+  }
+
+  onRadianceHeal(event: HealEvent) {
+    this.bonusRadianceHealing += calculateEffectiveHealing(event,SHINING_RADIANCE_INCREASE[this.conduitRank]);
```

Comment:
should probably just make `SHINING_RADIANCE_INCREASE[this.conduitRank]` a constant itself seems extra to keep accessing it from an array when its just going to grab the same number every time

Reply:
![image](https://user-images.githubusercontent.com/8739940/99112341-81b27380-25bb-11eb-81cf-9692afb5665f.png)

Reply:
beautiful